src/status_im/ui/components/invite/events.cljs

```diff
@@ -164,17 +165,23 @@
 
 (fx/defn starter-pack
   {:events [::starter-pack]}
-  [{:keys [db]}]
-  (let [contract (contracts/get-address db :status/acquisition)]
-    {::json-rpc/eth-call [{:contract   contract
-                           :method     "getDefaultPack()"
-                           :outputs    ["address" "uint256" "address[]" "uint256[]" "uint256[]"]
-                           :on-success #(re-frame/dispatch [::starter-pack-amount (vec %) (prn %)])}]}))
+  [{:keys [db]} contract]
+  {::json-rpc/eth-call [{:contract   contract
+                         :method     "getDefaultPack()"
+                         :outputs    ["address" "uint256" "address[]" "uint256[]" "uint256[]"]
+                         :on-success #(re-frame/dispatch [::starter-pack-amount (vec %) (prn %)])}]})
 
 (re-frame/reg-sub-raw
  ::starter-pack
  (fn [db]
-   (re-frame/dispatch [::starter-pack])
-   (make-reaction
-    (fn []
-      (get-in @db [:acquisition :starter-pack :pack])))))
+   (let [contract (get-in @db [:acquisition :contract])]
+     (re-frame/dispatch [::starter-pack contract])
+     (make-reaction
+      (fn []
+        (get-in @db [:acquisition :starter-pack :pack]))))))
+
+(re-frame/reg-sub
```

Comment:
we still don't have a decision on moving subs from subs.cljs

Reply:
Yes, but I wanted to keep the referral program code fully isolated from the app, as we do not know how it will evolve later, and how much time it will be there.